CONTRIBUTING.md

```diff
@@ -212,17 +255,13 @@ Recompile az_core with an HTTP client implementation like CURL to see sample sen
 i.e. cmake -DBUILD_CURL_TRANSPORT=ON ..
 ```
 
-When running cmake, next options can be used to change the output libraries/Pal/Samples:
-
-- `BUILD_CURL_TRANSPORT`: This option would build an HTTP transport library using CURL. It requires libcurl to be installed (vcpkg or globally). This option will make samples to be linked with this HTTP and be functional to send HTTP requests.
+## Running Tests and Samples
 
-```bash
-cmake -DBUILD_CURL_TRANSPORT=ON ..
-cmake --build .
-```
+### Unit tests
 
-- `UNIT_TESTING`: This option requires cmocka to be installed and it will generate unit tests for each project.
```

Comment:
Add a reference to the cmocka requirement

Reply:
it is mentioned in the #compiler-options section as:
><td>UNIT_TESTING</td>
<td>Generates Unit Test for compilation. When turning this option ON, cmocka is a required dependency for compilation.<br>After Compiling, use `ctest` to run Unit Test.</td>